src/module/auth/screen/WelcomeScreen.tsx

```
@@ -1,61 +1,201 @@
 
 export default function WelcomeScreen() {
   const navigation =
     useNavigation<NativeStackNavigationProp<AuthStackParamList>>();
 
+  // Animation values
+  const logoOpacity = useSharedValue(0);
+  const logoScale = useSharedValue(0.5);
+  const logoTranslateY = useSharedValue(50);
+  const animationTranslateY = useSharedValue(0);
+  const barAnimations = Array.from({ length: 5 }, () => ({
+    translateY: useSharedValue(0),
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)